guides/common/modules/proc_synchronizing-python-repositories.adoc

```diff
@@ -13,17 +13,16 @@ You can view the Python packages in the {ProjectWebUI} at *Content* > *Content T
 +
 The *Label* field is populated automatically based on the name.
 . From the *Type* list, select *python*.
-. In the *Upstream URL* field, enter the URL for the upstream content source, for example `\https://pypi.org`.
+. In the *Upstream URL* field, enter the URL for the upstream content source, for example `https://pypi.org`.
+. Optional: Unselect *Verify SSL* if you do not want {ProjectServer} to verify the SSL certificate of the upstream content source.
 . Optional: In the *Upstream Username* field, enter the user name for the upstream repository if required for authentication.
```

Comment:
This might be outside the scope of your PR, but I noticed that all these "Optional"s are not easy to read. Consider putting these into a bullet list with a lead-in sentence such as "You can set the following options:".

Reply:
To take it one step further, a lot of these steps are actually explained very clearly in the web UI itself. I'd say you can drop them from the docs.